### DISCUSSION TITLE
Recommendaiton for command-line utility for PNG lossy or lossless compression?

### DISCUSSION THREAD

**Discussion by ghost** [ImageMagick/ImageMagick#6252] [Help] [answered]
### ImageMagick version

7.0.8-Q16-HDRI

### Operating system, version and so on

Windows

### Description

Hi. Curious to know if anyone has any strong options about what command-line utility PNG compression tool to use. Looking for something a stable, proven, and maintained. I appreciate any suggestions. Thank you.

I have been experimented with compression with ImageMagick but not really getting good compression.

**Comment by snibgo** [comment, marked as answer]
I sometimes use `pngcrush`.

Do remember that PNG compression was designed for graphics, rather than photos. It doesn't give good compression for photos.

**Comment by ghost** [comment]
Okay that makes sense. I'm using photos. I will give pngcrush a try! Thank you.
